manifests/ota-deploy-state/configmap.yaml

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
Can remove ota-deploy-state and the commands for starting it etc, it isn't required for CE.

Reply:
Done.

Reply:
Done.